dask_cloudprovider/cloudprovider.yaml

```diff
@@ -119,6 +119,18 @@ cloudprovider:
     docker_image: "daskdev/dask:latest" # docker image to use
     bootstrap: true # It is assumed that the OS image does not have Docker and needs bootstrapping. Set this to false if using a custom image with Docker already installed.
 
+  ibm:
+    api_key: null
```

Comment:
Is there not a global location where IBM stores credentials? For example AWS stores things in `~/.aws/credentials` and we read it from there.

Reply:
No, according to the documentation it is not stored anywhere visible to the user. That is why it is necessary to add the API key manually.